pkg/fakerp/config.go

```diff
@@ -16,18 +16,23 @@ var supportedRegions = []string{
 }
 
 type Config struct {
-	SubscriptionID   string `envconfig:"AZURE_SUBSCRIPTION_ID" required:"true"`
-	TenantID         string `envconfig:"AZURE_TENANT_ID" required:"true"`
-	ClientID         string `envconfig:"AZURE_CLIENT_ID" required:"true"`
-	ClientSecret     string `envconfig:"AZURE_CLIENT_SECRET" required:"true"`
-	AADClientID      string `envconfig:"AZURE_AAD_CLIENT_ID"`
+	Username        string `envconfig:"AZURE_USERNAME"`
```

Comment:
We are not setting these in CI today, does this change mean we need to start doing that?

Reply:
Nevermind, we won't need them in CI

https://github.com/openshift/release/blob/859b37e14a88df9f88cf86b7af112ed6eee1c86b/ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml#L180